muni_portal/tests/api/test_councillor_group_page.py

```diff
@@ -32,6 +32,7 @@ def test_service_page_image(self):
             depth=2,
             live=True,
             overview="Test",
+            icon_classes=["test1", "test2"],
```

Comment:
why is this a list and not a string with a space?

Reply:
We can use `icon_classes="test1,test2"` as well, both options will be saved as comma-separated string.